.github/workflows/docs-pr.yml

```diff
@@ -77,13 +77,16 @@ jobs:
         run: |
           npm i wasm-opt -g
```

Comment:
This job could do with some cleaning up btw. Previously it would be building a couple of the JS packages so that it could codegen the reference docs however it seems like this is no longer the case.

We can then remove all the installation of tooling necessary for building the main workspace.